pkg/ctl/create/nodegroup.go

```diff
@@ -96,6 +98,41 @@ func filterNodeGroups(cfg *api.ClusterConfig) error {
 	return nil
 }
 
+func checkVersion(ctl *eks.ClusterProvider, meta *api.ClusterMeta) error {
```

Comment:
Ended up with a simple function here to begin with, we might want to find a better place for it later. I think we might actually promote nodegroup-level version field to reflect the real world, so I'm not too worried about generalising this.